v22.2/cockroach-debug-zip.md

```diff
@@ -146,7 +147,7 @@ Generate a debug zip file containing only log files:
 $ cockroach debug zip ./cockroach-data/logs/debug.zip --include-files=*.log
 ~~~
 
-### Redact sensitive information from the logs
+### Redact sensitive information
```

Comment:
This seems like a super important topic. I wonder if you are planning to do an in depth tutorial on this topic? It seems like it would be worthwhile and would probably be really good for SEO on our docs as well. Just a thought.

Reply:
Thanks, definitely a good call out! Eng is working on a list of explicit redactions and omissions presently. When ready, I'll be revisiting this page to add this list and flesh out the feature description accordingly.